projects/arlas-toolkit/assets/i18n/es.json

```diff
@@ -125,90 +99,138 @@
-	"Create the permission": "Crear permiso",
-	"Creating": "Creación en curso ...",
-	"Loading data of": "Carga de datos de '{{collection}}'",
-	"Search...": "Buscar...",
+	"Sign in": "Iniciar sesión",
+	"Please enter your password": "Introducir su contraseña",
+	"Forgot password?": "¿Ha olvidado su contraseña?",
+	"Login or password invalid": "Nombre de usuario/contraseña incorrectos",
+	"Login": "Iniciar sesión",
+	"No Account?": "¿No tiene cuenta?",
+	"x-axis": "Abscisa",
+	"to": "a",
+	"y-axis": "Ordenada",
+	"interval": "Intervalo",
+	"If you have an account, you will receive an email shortly": "Si tiene una cuenta, recibirá un correo electrónico en breve.",
+	"Send password reset email": "Recibir un correo electrónico de restablecimiento",
+	"Zoom to data": "Zoom a la extensión de los datos",
+	"See more": "Más información",
+	"Clear all filters": "Quitar todos los filtros",
+	"Actions": "Acciones",
+	"View": "Ver",
+	"Remove": "Quitar",
+	"Type": "Tipo",
+	"CSV": "CSV",
+	"Json": "Json",
+	"Important": "Importante",
+	"This tool allows you to get a docker command line to download your data": "Esta herramienta le permite recuperar una línea de comandos usando docker para descargar datos",
+	"Make sure you have": "Asegúrese de tener",
+	"installed before continuing": "instalado antes de continuar.",
+	"Collection": "Colección",
+	"Command line": "Línea de comandos",
+	"Target OS": "Sistema operativo de destino",
+	"Copy": "Copiar",
+	"Copied": "Se ha copiado",
+	"Close": "Cerrar",
+	"group": "Grupos",
+	"reading": "Permiso de lectura",
+	"writing": "Permiso de escritura",
+	"Share": "Compartir",
+	"Cancel": "Cancelar",
+	"Edit": "Editar",
+	"Duplicate": "Duplicar",
+	"Rename": "Cambia nombre",
+	"Delete": "Borrar",
+	"This will delete": "Esta acción borrará la configuración ",
+	"element and cannot be undone.": "y esta acción es irreversible.",
+	"New name": "Nuevo nombre",
+	"Add a new dashboard": "Crear un nuevo panel",
+	"Create": "Crear",
+	"Share dashboard": "Compartir panel",
+	"Change password": "Cambiar la contraseña",
+	"Old password": "Contraseña antigua",
+	"Old password does not match": "La contraseña antigua es incorrecta",
+	"Add a dataset": "Añadir un marcador",
+	"Manage datasets": "Gestionar los marcadores",
+	"Count": "Recuento de elementos",
+	"Bookmarks can be combined to create a new one containing the filters of all selected bookmarks.": "Los marcadores se pueden combinar para crear uno nuevo que contenga los filtros de todos los seleccionados.",
+	"The color next to the name matches the combination of fields and operations of the filters.": "El color junto al nombre coincide con la combinación de campos y operaciones de los filtros.",
+	"Filters on the same field are combined with an OR, whereas for different fields it is with an AND.": "Los filtros sobre el mismo campo se combinan con un OR, mientras que para campos diferentes se hace con un AND.",
```

Comment:
Los marcadores que comparten exactamente los mismos campos y operadores se combinan con un OR, de lo contrario se combinan con un AND